students/18th/team1/kookhyunpark/user/views.py

```diff
@@ -0,0 +1,49 @@
+import json
+import re
+
+from django.views import View
+from django.http  import JsonResponse, request
+
+from user.models import User
+
+class SignUpView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            email         = data['email']
+            phone         = data.get('phone', None)
+            full_name     = data.get('full_name', None)
+            user_name     = data.get('user_name', None)
+            password      = data['password']
+            date_of_birth = data.get('date_of_birth', None)
```

Comment:
필수필드, `NULL` 허용필드에 따른 접근 방식 다르게 적용 잘하셨습니다!! 👍👍👍
가독성을 위해 조금 위치를 변경할 수 있겠네요! 아래와 같이요ㅎㅎ
```suggestion
            email         = data['email']
            password      = data['password'] 
            phone         = data.get('phone', None)
            full_name     = data.get('full_name', None)
            user_name     = data.get('user_name', None)
            date_of_birth = data.get('date_of_birth', None)
```